packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -273,7 +280,48 @@ export class CXLMarketingNavElement extends LitElement {
         );
 
         // Populate.
-        contextMenu.items = this._createContextMenuItems(menuItem.children);
+        const contextMenuItems = this._createContextMenuItems(menuItem.children);
+        contextMenu.items = contextMenuItems;
+
+        const setDescription = async () => {
```

Comment:
Curious, would refactoring this -> "private" `_setContextMenuItemDescription()` class method make sense / is technically it feasible?

It's quite a large body in the middle of already a large method.

Reply:
This makes sense.